### PR TITLE
fix failing doc tests by avoiding parallelization at assembly level

### LINKED ISSUE #325: Doc tests are failing randomly
Doc tests are failing due to some random errors on different test executions.

See https://github.com/redis/NRedisStack/actions/runs/9971210131/job/27551791673?pr=275

### PR DESCRIPTION
**Pull request by atakavci**
Closes/Fixes #325

### CODE REVIEW

File: tests/Test.proj [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿<Project Sdk="Microsoft.Build.Traversal/2.0.24">
+  <ItemGroup>
+    <ProjectReference Include="**\*.*proj" />
+  </ItemGroup>
+</Project>
```
Comment (tishun) [inline tests/Test.proj:5]: Nit : seems to be missing trailing end line.

Btw do we run some formatter on the code?

**Review by tishun — APPROVED** [review]

**Review by vladvildanov — APPROVED** [review]

File: tests/Test.proj (hunk 1 above)
Comment (atakavci) [reply tests/Test.proj:5]: :point_right:  https://github.com/redis/NRedisStack/blob/master/.github/workflows/linter.yaml

Comment (tishun) [reply tests/Test.proj:5]: Wonder why this was not caught by the linter then

Comment (atakavci) [reply tests/Test.proj:5]: this file is something i introduced just to run tests one assembly at a time.
so it is not an artifact in sln file in root. 

also seem like `dotnet format` does not cover it anyway.  
```
dotnet format ./tests/Test.proj
Could not format '/home/atakavci/workspace/redis/nredisstack/NRedisStack/tests/Test.proj'. Format currently supports only C# and Visual Basic projects.
```
funny it is valid and accepted by `dotnet test`

Comment (tishun) [reply tests/Test.proj:5]: Sure, that's fine
I just wanted to make sure the liner is working, not a big issue, thanks for checking this

**Pull request merged**